ui/static/ui/js/listing_resources.jsx

```diff
@@ -430,6 +432,7 @@ define('listing_resources', ['react', 'jquery', 'lodash', 'utils'],
     var ListingPage = React.createClass({
       render: function() {
         return <div>
```

Comment:
What would an id conflict with? If classes are better, that's fine -- identification is my main concern.